docs/concepts/operator.md

```diff
@@ -153,10 +154,29 @@ Each one is described in more details below.
 - `.spec.deployment.extension.helm` and `.spec.deployment.extension.values` are used when creating the `ControllerDeployment` in the garden cluster.
 - `.spec.deployment.extension.policy` and `.spec.deployment.extension.seedSelector` define the extension's installation policy as per the [`ControllerDeployment's` respective fields](../extensions/controllerregistration.md#deployment-configuration-options)
 
-The extension controller can also be deployed in the runtime cluster to manage resources required by the `Garden` resource.
+##### Runtime
+
+The `extension` controller can be deployed in the runtime cluster to manage resources required by the `Garden` resource (e.g. `BackupBucket`, `DNSRecord`, `Extension`).
 Since the environment in the runtime cluster may differ from that of a `Seed`, the extension is installed in the runtime cluster with a distinct set of Helm chart values specified in `.spec.deployment.extension.runtimeValues`.
+If no `runtimeValues` are not given, the extension deployment for the runtime garden is considered superfluous and the deployment is skipped or uninstalled.
 This configuration allows for precise control over various extension parameters, such as requested resources, [priority classes](../development/priority-classes.md), and more.
 
+Besides the values configure in `.spec.deployment.extension.runtimeValues`, a runtime deployment indicator is merged into the values:
+
+```yaml
+gardener:
+  runtimeCluster:
+    enabled: true # indicates the extension deployment is responsible for the Garden cluster, e.g. for handling `BackupBucket`, `DNSRecord` and `Extension` objects.
+```
+
+As soon as a `garden` object is created and `runtimeValues` are configured, the extension is deployed in the runtime cluster. 
+
+##### Extension Registration
+
+Registering extensions for shoot clusters, e.g. cloud provider extensions, is mainly implemented through `ControllerRegistration`/`ControllerDeployment` resources.
+However, the `extension` controller helps to transform the information from an `Extension` object into these registration resources, so that `Extension`s become the single point of contact to enable such in the entire Garden landscape.
```

Comment:
You got me confused. Is `extension` something else compared to an `Extension`? Does the context (controller vs. object) make a difference?
Please clarify.

Reply:
Still, the question remains whether it is the `extension` controller or the `Extension` controller. I would lean towards an `Extension` controller as we usually capitalise the resource names, but maybe I am missing the big picture.

Reply:
From my point of view, the most confusing part is that there is an `extension controller` in the `operator` ([ref](https://github.com/gardener/gardener/blob/3330824bf66b579aa27ff318a26688cf149884fe/docs/concepts/operator.md?plain=1#L601)) which deploys an `extension controller` of a `gardener extension` (and an `admission controller`) ([ref](https://github.com/gardener/gardener/blob/3330824bf66b579aa27ff318a26688cf149884fe/docs/concepts/operator.md?plain=1#L145-L146)) in the same document.
If an extension controller is doing something ([ref](https://github.com/gardener/gardener/blob/3330824bf66b579aa27ff318a26688cf149884fe/docs/concepts/operator.md?plain=1#L159)), I'm wondering which one.
`Extension controller` vs. `extension controller` is an inaccurate spelling. I agree, we should capitalize it. 
However, it does not make the text any easier to understand. IMO we should not use `extension controller` as a reference to one part of a Gardener extension in this document.

Reply:
As discussed, I removed `extension controller` everywhere it mentioned the  extension controller of a "Gardener extension".
Now, there should be "`Extension` controllers" which all point to the controller in the `gardener-operator`.